src/node/interfaces.cpp

```diff
@@ -700,11 +699,9 @@ class ChainImpl : public Chain
     bool checkChainLimits(const CTransactionRef& tx) override
     {
         if (!m_node.mempool) return true;
-        LockPoints lp;
-        CTxMemPoolEntry entry(tx, 0, 0, 0, 0, false, 0, lp);
-        const CTxMemPool::Limits& limits{m_node.mempool->m_limits};
         LOCK(m_node.mempool->cs);
```

Comment:
The change in https://github.com/bitcoin/bitcoin/commit/e7e462eadc66e484ccb2f51967a0ad5f0d3bd41a has been reverted after the rebase.
```suggestion
        CTxMemPoolEntry entry(tx, 0, 0, 0, 0, false, 0, lp);
        LOCK(m_node.mempool->cs);
        std::string err_string;
        return m_node.mempool->CheckPackageLimits({tx}, entry.GetTxSize(), err_string);
```